app/views/surveys/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<p style="color: green"><%= notice %></p>
+
+<h1>Surveys</h1>
+
+<div id="surveys">
+<table>
+  <thead>
+    <tr>
+      <th>Survey Name</th>
+      <th>Survey Category</th>
+      <th>Survey Status</th>
+      <th></th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @surveys.each do |survey| %>
+      <tr>
+        <td><%= survey.survey_name %></td>
+        <td><%= survey.survey_category %></td>
+        <td><%= "Incomplete" %></td>
+        <td><%= link_to "Take Survey", select_language_survey_path(survey) %></td>
+      </tr>
+    <% end %>
+  </tbody>
+</table>
+</div>
+
+<%= link_to "New survey", new_survey_path %>
```

Comment:
This path should not be visible by default; anyone should not be able to create new surveys.

Reply:
Above route definition will solve this issue as well

Reply:
Dont think so, since we dont want regular patients to be able to create new surveys.